### PR TITLE
E7-V1 keyboard support

### PR DESCRIPTION
**Pull request by masterzen**
## Description

This change adds Exclusive latest keyboard the E7-V1, a 75% keyboard with an Atmega32U4 MCU, backlight and some rgb leds under the logo.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [x] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project (I hope).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

I wasn't able to test all the layouts, since I built mine with ANSI split backspace and the keyboard supports ANSI, ISO, split bs, 7u or 6.25 space...

This is my first keyboard contribution, let me know if you have any comments or improvements :)

### CODE REVIEW

File: keyboards/e7v1/config.h [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
```
Comment (vomindoraan) [inline keyboards/e7v1/config.h]: `IS_COMMAND` is no longer required to be defined if it uses the default value (LShift+RShift). Please remove lines 41–45 (inclusive) in this file.

**Comment by apocalypsemeow** [comment]
> Is this board related to the E6 keyboard, at all?

It's the same maker as the E6, Exlusive, but a different board by him.

**Comment by masterzen** [comment]
Thanks @drashna and @mechmerlin for your comments, I will fix that ASAP (and I should have read the contribution documentation first)...

**Comment by masterzen** [comment]
Hi @drashna and @mechmerlin ,

I think I've implemented all the requested changes. Thanks for pointing them.
Let me know if there's anything more I should add/amend.

Thanks,

**Comment by drashna** [comment]
Thanks!

**Pull request merged**